python/ql/lib/semmle/python/dataflow/new/internal/FlowSummaryImplSpecific.qll

```diff
@@ -25,6 +25,10 @@
  *
  * We do not support summaries of special methods via the special methods framework,
  * the summary would have to identify the call.
+ *
+ * We might, while we still extract the standard libraries, want to support flow summaries of
+ * extracted callables, so that we can model part of the standard libraries with flow summaries.
+ * For this to work, we have be careful with the enclosiong callable predicate.
```

Comment:
```suggestion
 * We might, while we still extract the standard library, want to support flow summaries of
 * extracted callables, so that we can model part of the standard library with flow summaries.
 * For this to work, we have be careful with the enclosing callable predicate.
```